templates/listing-page-filters.twig

```diff
@@ -0,0 +1,17 @@
+<div class="listing-page-filters">
+    <label for="categories" class="visually-hidden">
```

Comment:
You set `for="categories"`, but the `<select>` has `id="category"`. 
These IDs should match for accessibility.